pkgs/applications/gis/qgis/unwrapped.nix

```diff
@@ -120,36 +120,50 @@ in mkDerivation rec {
     ++ lib.optional withWebKit qtwebkit
     ++ pythonBuildInputs;
 
-  nativeBuildInputs = [ makeWrapper wrapGAppsHook cmake flex bison ninja ];
+  nativeBuildInputs = [ makeWrapper wrapQtAppsHook cmake flex bison ninja ];
 
   patches = [
     (substituteAll {
       src = ./set-pyqt-package-dirs.patch;
       pyQt5PackageDir = "${py.pkgs.pyqt5}/${py.pkgs.python.sitePackages}";
       qsciPackageDir = "${py.pkgs.qscintilla-qt5}/${py.pkgs.python.sitePackages}";
     })
+    ./fix-qsci.patch
   ];
 
+  #QT_QPA_PLATFORM_PLUGIN_PATH="${qt5.qtbase.bin}/lib/qt-${qt5.qtbase.version}/plugins/platforms";
+  QT_QPA_PLATFORM_PLUGIN_PATH="/nix/store/9j0acz9qqp1lygwif5jncpz8hsyfmylw-qtbase-5.15.9-bin/lib/qt-5.15.9/plugins/platforms";
+
   cmakeFlags = [
     "-DWITH_3D=True"
     "-DWITH_PDAL=TRUE"
+    "-DENABLE_TESTS=False"
+    "-DWITH_SERVER=False"
+    "-DWITH_QSCIAPI=False" # Scintilla API generation currently broken - see https://github.com/NixOS/nixpkgs/issues/248239
   ] ++ lib.optional (!withWebKit) "-DWITH_QTWEBKIT=OFF"
     ++ lib.optional withGrass (let
         gmajor = lib.versions.major grass.version;
         gminor = lib.versions.minor grass.version;
       in "-DGRASS_PREFIX${gmajor}=${grass}/grass${gmajor}${gminor}"
     );
 
-  dontWrapGApps = true; # wrapper params passed below
+  qtWrapperArgs = [
+    "--set QT_QPA_PLATFORM_PLUGIN_PATH $out/lib/qt-plugins/platforms"
```

Comment:
The path `$out/lib/qt-plugins/platforms` doesn't make a sense to me. What do you expect there ?

Reply:
Again an experiment to be removed.